internal/driver/driver.go

```diff
@@ -176,6 +178,12 @@ func applyCommandOverrides(cmd []string, v variables) variables {
 			v.set("nodecount", "80")
 		}
 	}
+
+	if outputFormat == report.Proto {
```

Comment:
Why did you keep `case "proto", "raw":` in the switch, isn't it redundant now?

Reply:
It's now redundant. Done.